interactive-opinion/src/css/main.scss

```diff
@@ -154,27 +164,32 @@ $gs-baseline: 12px;
     color: $comment-support-4;
 }
 
-.main__column--horizontal-bar:before {
-    content: '';
-    display: block;
-    border-bottom: 2px solid $comment-support-4;
-    margin: 0 .3rem;
-}
-
 .email-signup-header {
-    color: $neutral-1;
+    color: $neutral-7;
 }
 
 .email-signup-header__heading {
     @include fs-header(5);
-    padding: 0 .3rem;
+    padding: 6px 0px 4px .3rem;
+    font-size: 28px;
+    margin: 0;
 
     @include mq($from: tablet) {
         font-size: 36px;
-        margin-top: 8px;
+        margin-top: 2px;
     }
 }
 
+.email-signup-header__intro {
+    @include fs-header(2);
+    padding: 0 .3rem;
```

Comment:
for future reference, I would say use `px` everywhere since it all gets converted to `rem` anyway and that means everything is in the same (integer) unit